src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -187,8 +187,8 @@ public UserTypeClassesFormatDefinition()
         #endregion
         #region User Types - Records
         [Export(typeof(EditorFormatDefinition))]
-        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.RecordName)]
-        [Name(ClassificationTypeNames.RecordName)]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.RecordClassName)]
+        [Name(ClassificationTypeNames.RecordClassName)]
```

Comment:
Should the class name also update to `UserTypeRecordClassesFormatDefinition`? I don't belive it is ever surfaced to the user, so likely not an issue.

Reply:
Yeah, it's internal, so it's not an issue.